src/class.js

```diff
@@ -5,9 +5,20 @@
 // Add a method called `comparePasswords`.  `comparePasswords` should have a parameter
 // for a potential password that will be compared to the `password` property.
 // Return true if the potential password matches the `password` property.  Otherwise return false.
+class User {
+  constructor(options) {
+    this.email = options.email;
+    this.password = options.password;
+  }
 
+  comparePasswords(myPassword) {
+    if (this.password === myPassword) return true;
```

Comment:
Just do `return this.password === myPassword;` . It's automatically converted into a bool